github-lab-bundle-spec.md

```diff
@@ -142,16 +139,13 @@ legacy_display_options |          | array       | Elements to hide/show in ql-la
 
 ### Instructions
 
-attribute | required | type              | notes
---------- | -------- | ----------------- | -----------------------------------------
-type      | ✓        | enum              | [See list of valid types below]
-uri       | ✓        | string            | Paths to files in the bundle
+Place your instruction files under "instructions" folder. Name of the file MUST be same as the locale key.
 
-```yaml
-instruction:
-  type: html
-  uri: instructions/en.html
 ```
+my-lab/instructions/en.md
```

Comment:
Don't you need the `-instruction:` key before the files?

Reply:
Seems like the configuration section in qwiklabs.yaml is gone. This is an example of how the different locale files look like under the `instructions` folder.

Reply:
I didn't mean that as part of yaml, I'll clarify it